### ISSUE TITLE
Session's delete problem

### ISSUE THREAD

**Issue by afeena** [mushorg/tanner#189]
In develop branch I've got this error:
```2017-08-17 21:18 ERROR:aiohttp.server:log_exception: Error handling request
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/site-packages/aiohttp/web_protocol.py", line 421, in start
    resp = yield from self._request_handler(request)
  File "/usr/local/lib/python3.5/site-packages/aiohttp/web.py", line 303, in _handle
    resp = yield from handler(request)
  File "/usr/local/lib/python3.5/site-packages/Tanner-0.4.0-py3.5.egg/tanner/server.py", line 51, in handle_event
    data, self.redis_client
  File "/usr/local/lib/python3.5/site-packages/Tanner-0.4.0-py3.5.egg/tanner/session_manager.py", line 18, in add_or_update_session
    await self.delete_old_sessions(redis_client)
  File "/usr/local/lib/python3.5/site-packages/Tanner-0.4.0-py3.5.egg/tanner/session_manager.py", line 75, in delete_old_sessions
    self.sessions.remove(sess)
ValueError: list.remove(x): x not in list```

**Comment by rnehra01** [comment]
Could tell me how to reproduce it?

**Comment by afeena** [comment]
I made a lot of requests from different ip's. I have an idea why this problem may occurs: the delete process runs on request. When tanner tries to delete session, it can be already deleted by another deleting process of the other request (because delay between requests too small)

**Comment by nsmfoo** [comment]
I can just acknowledge that I'm getting the same errors in the tanner.err log. Sadly no more information to add at this time other then that.

**Comment by afeena** [comment]
@nsmfoo thanks, at least I know that it's not a problem of a particular machine :)

**Comment by viskey98** [comment]
@afeena  this issue is fixed by #231
